services/frontend-v3/src/i18n/fr_CA.json

```
@@ -276,6 +275,7 @@
   "profile.email": "Courriel",
   "profile.ex.feeder": "Je suis nommé dans le programme EX-feeder",
   "profile.not.ex.feeder": "Je ne suis pas nominé dans le programme EX-feeder",
+  "profile.private.ex.feeder": "Ex-Feeder est défini sur privé",
```

To keep it consistant and french translation change
```suggestion
  "profile.private.ex.feeder": "Le résultat d'EX-feeder est privé",
```